tests/functional/c/comparison_with_callable.py

```diff
@@ -58,3 +58,12 @@ def fake_property(self, prop):
 b = 786
 if a == b:
     pass
+
+
+def eventually_raise():
+    print()
+    raise Exception
+
+
+if a == eventually_raise:
```

Comment:
Could you add a little docstring about why this shouldn't raise? (I feel like I'm becoming the "docstring guy" 😄 )